synapse/push/mailer.py

```diff
@@ -191,11 +190,11 @@ def _fetch_room_state(room_id):
         multipart_msg.attach(html_part)
 
         logger.info("Sending email push notification to %s" % email_address)
-        # logger.debug(html_text)
 
-        yield sendmail(
+        yield self.hs.get_sendmail()(
```

Comment:
We generally try and pull out dependencies in the constructor, so that things fail at startup if there is a problem with initialising in dependencies

(I thought I made this comment, but GH forgot it :/)